content/docs/08-troubleshooting/05-ai-sdk-errors/api-call-error.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: APICallError
+description: Error thrown during a failed API call
+---
+
+# APICallError
+
+## When This Error Occurs
+
+The `APICallError` occurs when there's an issue during an API call made by the SDK. This can happen in various scenarios, such as:
+
+1. Network connectivity problems
+2. Server-side errors
+3. Rate limiting or request timeouts
+4. Invalid requests or authentication issues
```

Comment:
While this is generally true, it is really a misleading explanation.

This happens when a provider API returns with an error. Most of the time, this means that there API cannot be used in this way, e.g. invalid message structure. The user needs to carefully read the actual error message from the provider and figure out what to do.